notebooks/stable-fast-3d/stable-fast-3d.ipynb

```diff
@@ -0,0 +1,347 @@
+{
```

Comment:
<p>could you please add some details about model work and difference with previous TripoSR and maybe some diagram about model workflow? (<a href="https://stability.ai/news/introducing-stable-fast-3d" target="_blank">https://stability.ai/news/introducing-stable-fast-3d</a> <a href="https://stable-fast-3d.github.io" target="_blank">https://stable-fast-3d.github.io/</a>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2388/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Some model specific and comparation with TripoSR was added. Diagram and description in context of converting are in "details" in the part "Convert the model to OpenVINO IR".</p>